content/projects/java-specific/threads/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - topics/java-specific/concurrency
+  - topics/java-specific/functional-interface
+  soft:
+  - projects/java-specific/functional-interface
+ready: true
+submission_type: repo
+title: Java Threads
+---
+
+## The Chicken and the Farmer
```

Comment:
This doesn't say anything about the directory structure. Should it? 
If the goal here is to keep it open-ended that's ok, but we might at least want to specify that they use gradle

Reply:
Need to get some clarity from Ruddy here

Reply:
the directory structure is nothing special so maybe we go with
```
├── build.gradle
├── gradle
│   └── wrapper
│       ├── gradle-wrapper.jar
│       └── gradle-wrapper.properties
├── gradlew
├── gradlew.bat
├── settings.gradle
└── src
    └── main
        └── java
             └── ChickenFarm.java <------ names are important
```